yb-voyager/src/srcdb/postgres.go

```diff
@@ -415,11 +417,53 @@ func (pg *PostgreSQL) FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sql
 }
 
 func (pg *PostgreSQL) GetTableColumns(tableName *sqlname.SourceName) ([]string, []string, []string) {
-	return nil, nil, nil
+	var columns, dataTypes, dataTypesOwner []string
+	query := fmt.Sprintf(`SELECT a.attname AS column_name, t.typname AS data_type, rol.rolname AS data_type_owner 
+						FROM pg_attribute AS a JOIN pg_type AS t ON t.oid = a.atttypid 
+						JOIN pg_class AS c ON c.oid = a.attrelid 
+						JOIN pg_namespace AS n ON n.oid = c.relnamespace 
+						JOIN pg_roles AS rol ON rol.oid = t.typowner 
+						WHERE c.relname = '%s' AND n.nspname = '%s' AND a.attname NOT IN ('tableoid', 'cmax', 'xmax', 'cmin', 'xmin', 'ctid');`, tableName.ObjectName.Unquoted, tableName.SchemaName.Unquoted)
+	rows, err := pg.db.Query(context.Background(), query)
+	if err != nil {
+		utils.ErrExit("error in querying(%q) source database for table columns: %v\n", query, err)
```

Comment:
As a general practice, we shouldn't be using utils.ErrExit and rely on returning error objects. Quite a bit of the existing code is written with utils.ErrExit, but let's try our best to not add more. 

If possible, fix this func and GetColumnsWithSupportedTypes in the interface.

Reply:
Done